comp/core/settings/settingsimpl/settingsimpl.go

```diff
@@ -142,6 +142,30 @@ func (s *settingsRegistry) GetFullConfig(namespaces ...string) http.HandlerFunc
 	}
 }
 
+func (s *settingsRegistry) GetFullConfigBySource() http.HandlerFunc {
+	return func(w http.ResponseWriter, _ *http.Request) {
+		settings := s.config.AllSettingsBySource()
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Because the body is json formatted, it should be good to add the `Content-Type‏` header.
```suggestion
w.Header().Set("Content-Type", "application/json")
```